### DISCUSSION TITLE
表格识别中输出的中文文字识别有错误，如何解决？

### DISCUSSION THREAD

**Discussion by YingxuanW** [PaddlePaddle/PaddleOCR#13831] [Q&A]
请问各位大佬，pp-structure中表格识别输出结果文字识别有错误，该如何解决？微调文字识别模型？还是更改用其他文字识别模型替代呢？如果要替代的话，该怎么替代？paddle框架和transformer兼容吗？

**Comment by GreatV** [comment]
> 1. pp-structure中表格识别输出结果文字识别有错误，该如何解决？微调文字识别模型？

可以试试看

> 2. 还是更改用其他文字识别模型替代呢？

可以试试高精度模型 参考 https://paddlepaddle.github.io/PaddleOCR/ppocr/model_list.html#pp-ocrv4202381

> 3. 如果要替代的话，该怎么替代？paddle框架和transformer兼容吗？

应该不太兼容，容易出现各种问题。

**Reply by YingxuanW** [reply]
我换了高精度模型，然后报错了，是有哪些config也需要修改吗？麻烦您了！

**Reply by GreatV** [reply]
需要提供一个最小可复现的demo，我们才能给出建议

**Reply by YingxuanW** [reply]
ExternalError: CUDNN error(9), CUDNN_STATUS_NOT_SUPPORTED. 
      [Hint: 'CUDNN_STATUS_NOT_SUPPORTED'.  The functionality requested is not presently supported by cuDNN.  ] (at ../paddle/phi/kernels/fusion/gpu/fused_conv2d_add_act_kernel.cu:618)
      [operator < fused_conv2d_add_act > error]
    原来的低精度模型能出结果，换了高精度就报这个错，可能是环境不适配？

**Reply by GreatV** [reply]
是不是显卡显存太小了，换成cpu试试

**Reply by YingxuanW** [reply]
cuda_visible_devices=3,4,5,6,7  5卡a100 40g

**Reply by GreatV** [reply]
可以更新一下paddle到3.0beta 或者2.6.1 https://www.paddlepaddle.org.cn/install/quick

**Reply by YingxuanW** [reply]
好的，我的paddle版本是3.0beta。然后，我刚刚用cpu确实能跑，感谢！

**Comment by GreatV** [comment]
确实会有问题 参考 https://github.com/PaddlePaddle/Paddle/issues/60540

**Reply by YingxuanW** [reply]
感谢耐心回复！
